File: 100-Days-Series/Day 61/PalindromicPatternWithNumber.js

// Arrow Functiont to print Palindromic Pattern with number;
function getPrintPalindromic(rows){
    for(let i=1;i<=rows;i++){
        line="";
        // Collect Spaces
        for(let j=1;j<=rows-i;j++){
            line+="  ";
        }
        // Collect Increasing numbers
        for(let j=1;j<=i;j++){
            line+=j+" ";
        }
        // Collect Decreasing numebrs
        for(let j=i-1;j>=1;j--){
            line+=j+" ";
        }
        console.log(line);
    }
}
//Let's Start here
let rows=5;
getPrintPalindromic(rows);

File: 100-Days-Series/Day 61/PalindromicPatternWithNumber.py
# Function to print Palindromic Pattern with number
def getPrintPalindromic(rows):
    for i in range(1,rows+1,1):
        # Print Spaces
        for j in range(0,rows-i,1):
            print(end="  ")
        # Print Increasing numbers
        for j in range(1,i+1,1):
            print(j,end=" ")
        # Print Decreasing numbers
        for j in range(i-1,0,-1):
            print(j,end=" ")
        print()

# Let's Start Here
rows=int(input("Enter the number of rows : "))
getPrintPalindromic(rows)
